sim/scenarios/helper/quic-point-to-point-helper.cc

```diff
@@ -14,6 +14,9 @@ void QuicPointToPointHelper::SetQueueSize(StringValue size) {
 
 NetDeviceContainer QuicPointToPointHelper::Install(Ptr<Node> a, Ptr<Node> b) {
   NetDeviceContainer devices = PointToPointHelper::Install(a, b);
+  // capture a pcap of all packets
+  EnablePcap("/logs/trace_node_left.pcap", devices.Get(0), false, true);
+  EnablePcap("/logs/trace_node_right.pcap", devices.Get(1), false, true);
```

Comment:
This is on the server side, right?

Reply:
This is on both sides of the p2p link.

Reply:
Cool, thanks! I can't keep these device IDs straight.

Reply:
We might want to rename the log files. Let's get this merged and move on from there.